components/ThemeManager.vue

```diff
@@ -0,0 +1,147 @@
+            </p>
+            <SvgoAccordionArrow
+                id="accordion"
+                role="img"
+                title="accordion arrow"
+                :class="colorThemeAccordionIsClosed ? 'w-6 h-6 transition duration-300 fill-primary' : 'w-6 h-6 transition duration-300 fill-primary -rotate-180'"
+            />
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { SvgoAccordionArrow } from '#components'
+
+const { t } = useI18n()
+
+const currentTheme = ref('original')
+
+const isDarkMode = ref(false)
+
+const themes = reactive([
+    {
+        themeId: 'original',
+        dotColor: '#0EB0C0',
+        themeName: 'Original',
+        isSelected: true,
+        isDarkMode: isDarkMode
+    },
+    {
+        themeId: 'coral',
+        dotColor: '#ED6C5A',
+        themeName: 'Coral',
+        isSelected: false,
+        isDarkMode: isDarkMode
+    },
+    {
+        themeId: 'violet',
+        dotColor: '#A45D9A',
+        themeName: 'Violet',
+        isSelected: false,
+        isDarkMode: isDarkMode
+    },
+    {
+        themeId: 'accessible-high-contrast',
+        dotColor: '#006872',
+        themeName: 'High Contrast',
+        isSelected: false,
+        isDarkMode: isDarkMode
+    },
+    {
+        themeId: 'accessible-red-green',
+        dotColor: '#007BFF',
+        themeName: 'Red-Green Color Blindness',
+        isSelected: false,
+        isDarkMode: isDarkMode
+    }
+])
+
+const colorThemeAccordionIsClosed = ref(true)
+
+function toggleThemeVisibility() {
+    colorThemeAccordionIsClosed.value = !colorThemeAccordionIsClosed.value
+}
+
+function toggleLightDarkMode(returnedDarkModeValue) {
+    isDarkMode.value = returnedDarkModeValue
+    localStorage.setItem('isDarkMode', `${isDarkMode.value}`)
+    setTheme(currentTheme.value, isDarkMode.value)
+}
+
+function setTheme(newTheme: string, darkModeValue: boolean) {
+    // ---Remove selected from previous theme
+    const selectedTheme = themes.find(theme => theme.isSelected)
+    selectedTheme.isSelected = !selectedTheme.isSelected
+
+    // ---Set theme to selected---
+    const identifiedTheme = themes.find(theme => theme.themeId === newTheme)
+
+    identifiedTheme.isSelected = !identifiedTheme.isSelected
+
+    document.documentElement.classList.remove(
+        'theme-original',
+        'theme-coral',
+        'theme-violet',
+        'theme-original-dark',
+        'theme-coral-dark',
+        'theme-violet-dark',
+        'theme-accessible-high-contrast',
+        'theme-accessible-red-green',
+        'theme-accessible-high-contrast-dark',
+        'theme-accessible-red-green-dark'
+    )
+
+    if (!darkModeValue) {
+        document.documentElement.classList.add(`theme-${newTheme}`)
+    } else {
+        document.documentElement.classList.add(`theme-${newTheme}-dark`)
+    }
+    localStorage.setItem('theme', newTheme)
+    localStorage.setItem('isDarkMode', `${darkModeValue}`)
```

Comment:
I'm not sure you actually need this since you're setting in the dark mode toggle already